code/modules/clothing/under/jobs/security.dm

```diff
@@ -155,4 +155,15 @@
 	item_color = "brig_phys"
 	permeability_coefficient = 0.50
 	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 10, rad = 0)
+	flags = ONESIZEFITSALL
+
+//Pod Pilot
+/obj/item/clothing/under/rank/security/podpilot
+	desc = "Suit for your regular pod pilot."
+	name = "pod pilot's jumpsuit"
+	icon_state = "pod_pilot"
+	item_state = "pod_pilot"
+	_color = "pod_pilot"
+	permeability_coefficient = 0.50
+	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)

```

Comment:
You don't actually need to define this, since it's the same as the regular security jumpsuit...
All of the above ones with identical armor have it defined, oh god why.